ONSdigital/census-rh-ui
# CR-1194 NISRA CE4 code

Signed-off-by: Paul-Joel <paul.joel@ons.gov.uk>

# Motivation and Context
NI CE Manager UAC's start 'CE4' but are not dealt with by us. Inform the user

# What has changed
Code checks submitted UAC to see if it starts 'CE4'. If 'CE4' and on /ni user gets new error page, directing them to NISRA Communal Portal. If /en or /cy, user gets error for 'code for Northern Ireland'
Added unit tests to check, and updated incorrect existing test for missing uac

No Cucumber updates required

# How to test?
Enter a UAC beginning CE4 on all 3 start pages, and check appropriate error is shown

# Links
https://collaborate2.ons.gov.uk/jira/browse/CR-1194
